### DISCUSSION TITLE
question about compute vision

### DISCUSSION THREAD

**Discussion by albertopatrin** [albertopatrin/vision_classifier#1] [Q&A] [answered]
how can iearn faster visual computing ? `print("hash faster way")`

**Comment by Huelerssey** [comment, marked as answer]
u can patric every single day bro, and use videos about for learn faster. i personally recomend: https://www.youtube.com/watch?v=GwH3N630pwA&list=PLuv3GM6-gsE1Biyakccxb3FAn4wBLyfWf
